### DISCUSSION TITLE
利用AI來種茶-智慧農業

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#150] [General]
近期敏實科技大學人工智慧學院院長林文燦，利用AI人工智慧技術，對生長環境、水分、養分以及天候等進行監測，並以大數據來分析不同的生產條件。而因為利用科技讓每一個步驟都做到最好，天候狀況等都控制得宜，並有應對方法，因此種出來的茶葉自然又香又順口。希望不只有烏龍茶，期待以後能培養出更多品種的茶葉，有機會還可以外銷!

文章來源:https://tw.news.yahoo.com/ai-%E6%99%BA%E6%85%A7%E8%BE%B2%E6%A5%AD%E5%B1%95%E7%A5%9E%E9%80%9A-%E5%86%AC%E8%8C%B6%E7%83%8F%E9%BE%8D%E5%8B%87%E5%A5%AA%E9%A0%AD%E7%AD%89%E7%8D%8E-112848539.html

**Comment by bonny123865** [comment]
我覺得幫助茶農栽培出好茶，不僅是對於農業有好的成果，也讓未來產業技術結合有更多可能性！

**Comment by Jay-722** [comment]
雖然現在越來越少人喝茶，但是說不定可以跟手搖產業結合

**Comment by pocper1** [comment]
也可以預測颱風來對茶葉的損失！

**Comment by minniewei** [comment]
會不會在未來我們不需要有農夫可以全自動化(期待...)

**Comment by briankuo123** [comment]
未來的人類生活是不是都會靠AI自動生產，那我們人類以後不知道都會做甚麼。會不會可以整天躺在家裡看電影。

**Comment by wh20502** [comment]
我還滿好奇是用ai在輔導什麼，搞不好可以用ai來輔導種花，監測花的生長，開得更漂亮之類的(觀光區或是花市)

**Comment by 110409538** [comment]
感覺很多職業都會被AI取代...

**Comment by yasklop** [comment]
感覺很不錯，好想喝喝看！

**Comment by ghost** [comment]
但又衍生一個問題 "成本" 會擔心茶葉價格過高而較少人買單

**Comment by Shiba010** [comment]
AI在這方面感覺真的可以做的比人類好。

**Comment by jack90325** [comment]
將智能運用在農業上真的是十分方便，分析數據來將環境改成最適合作物生長的環境

**Comment by ayakasanma** [comment]
那麼鄉下老人家們要做什麼QQ他們一把年紀沒辦法與時俱進卻還要跟AI競爭，真是太殘酷了

**Comment by LanningYang** [comment]
個人覺得茶園幅地廣闊 AI 僅能收集數據 讓農民能更加掌控採收時間

**Comment by lx3211** [comment]
現在AI還只是輔助，未來不久候感覺就可以獨挑大梁種菜ㄌXD

**Comment by NcuJerryKao** [comment]
AI若能種茶 將來又有一部分的職業被取代
真是惶恐
想來AI能畫畫 畫家被取代
能做音樂 能自駕車
能解題 且寫出詳解 將來連補教界都會天翻地覆

**Comment by BingRong1217** [comment]
科技在很多真的造福人類蠻多的，雖然可能也會帶來一些反效果？

**Comment by Yilin20021016** [comment]
我好喜歡梨山茶阿 有一種獨特的香味 希望可以用AI多種點

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by awkward-willy** [comment]
透過 AI 科技能幫助農民針對難以控制的氣候做出最合適的選擇，進而減少因天災或判斷失誤等產生的損失，也能增進產量，提升品質，一舉數得。在未弄清楚基因改造的副作用前，或許智慧農業是農業轉型更好的選擇。

**Comment by boson13579** [comment]
AI幫助茶農偵測、控制土壤濕度、酸鹼度、紅土含量及預防過於強烈的天氣變化，不只減少了茶農們的損失，也減少了一般茶業對於人力的投入。

**Comment by yjpxi** [comment]
我覺得利用ai來種植茶葉可以讓茶葉的品質變高，高品質的茶葉可以進入國際市場，增加外銷機會，提升台灣經濟效益。但是如果要普及ai種植茶葉可能會對小農戶造成財產的負擔，因為引入和維護這些先進技術需要高昂的成本。
